orchestrator/gravity_utils/src/types/config.rs

```diff
@@ -0,0 +1,32 @@
+use serde_derive::Deserialize;
+use std::fmt;
+use std::str::FromStr;
+
+/// The various possible modes for relaying
+#[derive(Debug, Deserialize, PartialEq, Copy, Clone)]
+pub enum RelayerMode {
+    /// Always relay batches, profitable or not
+    AlwaysRelay,
+    /// Use private API to fetch the price data feed for the cost estimation
+    Api,
+    /// Use file to fetch the token price for the cost estimation
+    File,
+}
+
+impl fmt::Display for RelayerMode {
```

Comment:
why impl `Display` while the result is no difference with `Debug` ?

Reply:
got this error if I tried to remove

```
error[E0277]: `RelayerMode` doesn't implement `std::fmt::Display`
  --> relayer/src/main.rs:84:37
   |
84 |     info!("Relayer using mode: {}", mode);
   |                                     ^^^^ `RelayerMode` cannot be formatted with the default formatter
   
```

Not sure if there is a better way

Reply:
use` (".... {:?}", mode)`